src/vdaf/hits.rs

```diff
@@ -229,8 +242,11 @@ impl<I: Idpf<2, 2>> Vdaf for Hits<I> {
 }
 
 /// An input share for the heavy hitters VDAF.
+#[derive(Debug, Clone, Serialize, Deserialize)]
 pub struct HitsInputShare<I: Idpf<2, 2>> {
     /// IDPF share of input
```

Comment:
nit: Add a blank line `//` under the docucomment. (I'm not sure if rust has a convention for docucomments followed immediately by a normal comment.)